enhancements/cluster-logging/cluster-logging-otel-support.md

```diff
@@ -0,0 +1,242 @@
+* As a developer, I want users to be able to query log records using OTEL or [ViaQ](https://github.com/openshift/cluster-logging-operator/blob/release-5.9/docs/reference/datamodels/viaq/v1.adoc)
+attributes: log type, namespace, pod name, container name to facilitate LokiStack's transition from using ViaQ to OTEL
+* As a developer, I want Red Hat LokiStack to provide a native OTLP ingestion point
+so that other Red Hat products can forward observability signals using OTEL and rely upon OpenShift Logging tenancy and RBAC integration
+
+The administrator role is any user who has permissions to deploy the operator and the cluster-wide resources required to deploy the logging components.
+The developer role is any member of the Red Hat OpenShift Logging team.
+
+### Goals
+
+* Provide an in-cluster log search and alerting experience based on [OpenTelemetry Semantic Conventions](https://opentelemetry.io/docs/specs/semconv/)
+* Provide an OTLP-native log ingestion for third party trusted/verified collectors.
+* Provide native zero-effort OpenShift Logging Tenancy and RBAC integration to third-party trusted/verified collectors
+* Provide a transition period for users to continue to use the ViaQ model until they must modify their logQL queries to use OTEL
+
+### Non-Goals
+
+* Entirely replace the [ViaQ](https://github.com/openshift/cluster-logging-operator/blob/release-5.9/docs/reference/datamodels/viaq/v1.adoc) data model with OpenTelemetry Semantic Conventions.
+* In-cluster log storage for public, non-verified log ingestion.
+
+## Proposal
+
+In order to realize the goals of this enhancement:
+
+* Modify **cluster-logging-operator** to allow writing to **LokiStack** over ViaQ(the default) or OTLP based upon a **ClusterLogForwarder** annotation
+* Modify the vector log collector deployment to write to **LokiStack** over OTLP and additionally include select ViaQ attributes
+* Update the **loki-operator** to deploy v3.x are better of Loki to provided the an OTLP endpoint
+* Update the **loki-operator** to provide sane defaults to the new LokiStack OTEL API for server-side indexing of OTEL attributes to improve query performance
+* Modify the **logging-view-plugin** to use OTEL attributes that correspond to ViaQ *log_type*, *log_source*, *kubernetes.container_name*, *kubernetes.namespace_name*, *kubernetes.pod_name*
+
+### Workflow Description
+
+The following workflow describes deployment of a full logging stack to collect and forward logs to a Red Hat managed log store, **LokiStack**, using OTLP.
+
+**cluster administrator** is a human responsible for:
+
+* Managing and deploying day 2 operators
+* Managing and deploying an in-cluster LokiStack
+* Managing and deploying a cluster-wide log forwarder
+
+**cluster-observability-operator** is an operator responsible for:
+
+* managing and deploying observability operands (e.g. LokiStack, ClusterLogForwarder, Tracing) and console plugins (e.g console-logging-plugin)
```

Comment:
```suggestion
* managing and deploying observability operands and console plugins (e.g console-logging-plugin)
```

Removing the examples for operands here to avoid confusion. As per now it is not possible to install CLF/LokiStack/Tracing via COO. This is a future outcome and has not materialized yet.